cfssl.go

```diff
@@ -94,6 +98,10 @@ func registerFlags() {
 	cfsslFlagSet.StringVar(&Config.domain, "domain", "", "remote server domain name")
 	cfsslFlagSet.StringVar(&Config.ip, "ip", "", "remote server ip")
 	cfsslFlagSet.StringVar(&Config.remote, "remote", "", "remote CFSSL server")
+	cfsslFlagSet.BoolVar(&Config.short, "short", false, "so not run long/intensive scans")
```

Comment:
It was intended as "don't run long and/or intensive scans". But, preferably we won't have those, or you can select to not run the entire suite. The `short` argument has been removed, and and replaced with a `verbose` output argument.